### ISSUE TITLE
Need to update FFmpegWriter example at http://www.scikit-video.org/stable/io.html

### ISSUE THREAD

**Issue by RaviSoji** [scikit-video/scikit-video#41]
The FFmpegWriter example (in section titled "Writing") at http://www.scikit-video.org/stable/io.html appears to be out of date. The updated example I found (at http://www.scikit-video.org/dev/examples/io.html#ioexamplevideo) does run for me, however. 

The former link was significantly more visible/accessible through Google search for me, so it might be worth pushing the change as it would make a basic feature more accessible to new users or coders.

**Comment by beyondmetis** [comment]
I completely agree. I've been trying to fix the documentation builder circleci script to generate the actual latest dev documentation toward making a new release, but I keep running into dependency issues. Any help in fixing that is welcome, since after that I can push a new release version.

**Comment by beyondmetis** [comment]
I pushed the change from dev -> v1.1.8. This should solve half the problem. The reading/writing video section is still seriously out of date.

**Comment by Meixu-Chen** [comment]
The documentation website has been updated as part of the v1.1.12 release. The FFmpegWriter examples at http://scikit-video.org/stable/io.html now reflect the current API.
